docs/en/introduction/io_processing.md

```diff
@@ -2,60 +2,60 @@
+JuiceFS supports sequential reads and random reads (including mmap-based random reads). During read requests, the object corresponding to the block is completely read through the `GetObject` API of the object storage, or only a certain range of data in the object may be read (e.g., the read range is limited by the `Range` parameter of [S3 API](https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetObject.html)). Meanwhile, prefetching is performed (controlled by the [`--prefetch`](../reference/command_reference.md#mount) option) to download the complete data block into the local cache directory, as shown in the `blockcache` write speed in the second stage of the above metrics figure. This is very good for sequential reads as all cached data is utilized, maximizing the object storage access efficiency. The dataflow is illustrated in the figure below:
 
 ![](../images/internals-read.png)
 
-Although prefetching works well for sequential read, it might not be so effective for random read on large files, rather, it can cause read amplification and frequent cache eviction, consider disabling prefetching using `--prefetch=0`. It is always hard to design cache strategy for random read scenarios, one possible solution is to increase the cache size so that all data can be cached locally, while another solution could be disabling cache altogether (`--cache-size=0`), so that all reads are served directly by object storage, you'll be needing a high performance object storage service using this method.
+Although prefetching works well for sequential reads, it might not be so effective for random reads on large files. It can cause read amplification and frequent cache eviction. Consider disabling prefetching using `--prefetch=0`. It is always hard to design cache strategy for random read scenarios. Two possible solutions are increasing the cache size to store all data locally or completely disabling the cache (`--cache-size=0`) and relying on a high-performance object storage service.
 
-Reading small files (smaller than block size) is much easier. The entire file can be read in a single request. Since small files are cached locally along the way when written to the object storage, future read will be very fast because they are already cached locally. For example, `juicefs bench` behaves precisely this way, so expect an impressive performance.
+Reading small files (smaller than the block size) is much easier. The entire file can be read in a single request. Since small files are cached locally during the write process, future reads are very fast due to the local caching. This approach provides impressive performance, as demonstrated by `juicefs bench`.
```

Comment:
bench 命令是允许指定 block size 的，这一句的表述已经不再准确（也许从前 bench 命令只写小文件）

```suggestion
Reading small files (smaller than the block size) is much easier because the entire file can be read in a single request. Since small files are cached locally during the write process, future reads are fast.
```